hles/extraction/src/main/scala/org/broadinstitute/monster/dap/EnvironmentExtractionPipeline.scala

```diff
@@ -22,6 +22,7 @@ object EnvironmentExtractionPipeline extends ScioApp[Args] {
   )
 
   val subdir = "environment"
+
```

Comment:
My IntelliJ keeps making this change for me and it has been so annoying.

Reply:
i don't know that this is intelliJ, since i don't use it - i think this is an automated change by `scalafmt`